timbermill-server/src/test/java/com/datorama/oss/timbermill/TimberLogLocalTest.java

```diff
@@ -27,6 +27,7 @@ public static void init(){
 
     @AfterClass
     public static void tearDown(){
+        pipe.close();
```

Comment:
Don't do it, it causes problems

Reply:
it fails tests otherwise, since it tries to reschedule some of the jobs.
I'll try to re-exemint it